src/Servers/HttpSys/src/MessagePump.cs

```diff
@@ -138,24 +136,34 @@ private void ActivateRequestProcessingLimits()
         {
             for (int i = _acceptorCounts; i < _maxAccepts; i++)
             {
-                ProcessRequestsWorker();
+                _ = ProcessRequestsWorker();
```

Comment:
As Roslyn will give a CS4014 error about an unawaited task without it

> Error CS4014
> Because this call is not awaited, execution of the current method continues before the call is completed. Consider applying the 'await' operator to the result of the call.